src/polyswarm/client/bundle.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import os
+
+import click
+
+from polyswarm.client import utils
+from polyswarm_api import settings, resources
+
+logger = logging.getLogger(__name__)
+
+
+@click.group(short_help='Interact with creating or downloading a password protected bundle archive of sample/s.')
+def bundle():
+    pass
+
+
+@bundle.command('create', short_help='Create a bundle for an instance/s.')
+@click.option('-i', '--instance-id', type=click.STRING, multiple=True,
+              help='The ID of an instance to include in the bundle archive.')
+@click.option('-n', '--archive-name', type=click.STRING, help='Name of the archive that will be created.')
```

Comment:
I would use a positional argument instead for the file name, using the `clicks.Path` type maybe, like: https://github.com/polyswarm/polyswarm-cli/blob/develop/src/polyswarm/client/scan.py#L30C1-L30C79

Reply:
Hey @mrsarm I thought about this some more as I was working on it and I think the option may be better here. Because this arg isn't required by the API, and the `download` command below accepts a `click.Path` option for where to download the zip to.  This archive-name is more of a 'default' name for when someone downloads this bundle without setting a name/path themselves.